.travis.yml

```diff
@@ -17,3 +17,10 @@ deploy:
   script: _scripts/deploy.sh
   on:
     branch: master
+notifications:
+  webhooks:
+    urls:
+      - secure: "k9yfhsC6oqlOv7pgd31qA8Nk1PykhJX4ojko2ljnH7XonjHRrGFhh+5eSfnqNxUoETeJorazGQgku9loUPquAFZBtO2tpjf0nb1WMU5fuVGYwXu8Fi+ddD8XusufJuHh9gt3cMHhzB26Baz5pfKgTt3g/GGNKAkf1Pg/pcv5Ulfud9dvLVef0yB4xph5W100YfP8YcFTPppNlXwDCmMXacQHVWlDHLpF2wieMqpb6w+tPt+H4iV1I5ggJjpjo85czr2wF2W+U+L8Rd5cEMNVzC2PnkPtX5TcIPwFOnUhkrN5Ir2Er9Aztf868CYoE7oHysaSM7OgavOFo70x7MKwSRsnMKnVgx3q0GRHBLcffQi8ZqG5C4Lmdn70n/FoCLSsOKHnp0u7HeMDRvFpVnuWw5JndFXhU3cgVgCPdj8AN5k6VQKwfKafZit5EIr6muI1czS36jVCORjTNo2X6kDOH2UJe3GWNYapD9B8poZUPbOJzZ8wBgX3zU6s93JTChL6cTXcN24Bzs1O/bwLJSnGjeI+WiPLc8iSRpiH1a1VLGwL1D2P0ZHcVy7rPqr79WEqslFI+c2L9q991/ejcOWjzI1NjGPsz0kti1Mv3ckT+1IeQpJ9jM2Tg+3rO8kwEpJHNBil6uhtb/f+E5ltQg+q3gMv6SDuAe0bKgnyVvbbw40="
```

Comment:
Simply for my own edification... I am trying to understand this line.  I presume that the encrypted value is a pre-signed URL for triggering a Jenkins job?